recruitment-challenges/large-file-reading-challenge/README.md

```diff
@@ -0,0 +1,38 @@
+# Large file reading challenge
+
+Welcome in the recruitment challenge.
+Write an application that, at the endpoint specified by you, returns the monthly average temperatures for a given city in the format (here will be the format).
+
+## Assumptions
+
+- CSV file with data is no less than 3GB in size.
+- The file represents temperature measurements in the format city;yyyy-mm-dd;temp
+- The content of the source file may change during the application's running
+
+## Example source file
+[example_file.csv](example_file.csv)
+
+
+## Example response
```

Comment:
examples show the yearly average temperature while the task description mentioned about monthly average temp